test/input/yaml/Florence_Benchmark.yaml

```diff
@@ -0,0 +1,107 @@
+        waterbody: "NHDWaterbodyComID"
+        tw: "TopWdth"  # TODO: rename to `topwidth`
+        twcc: "TopWdthCC"  # TODO: rename to `topwidthcc`
+        musk: "MusK"
+        musx: "MusX"
+        cs: "ChSlp"  # TODO: rename to `sideslope`
+    waterbody_null_code: -9999
+    terminal_code: 0
+    driver_string: NetCDF
+    layer_string: 0
+#waterbody parameters and assignments from lake parm file
+waterbody_parameters:
+    level_pool:
+        #WRF-Hydro lake parm file
+        level_pool_waterbody_parameter_file_path: "../../test/input/florence_933020089/DOMAIN/LAKEPARM.nc"
+        level_pool_waterbody_id: lake_id
+        level_pool_waterbody_area: LkArea
+        level_pool_weir_elevation: WeirE
+        level_pool_waterbody_max_elevation: LkMxE
+        level_pool_outfall_weir_coefficient: WeirC
+        level_pool_outfall_weir_length: WeirL
+        level_pool_overall_dam_length: DamL
+        level_pool_orifice_elevation: OrificeE
+        level_pool_orifice_coefficient: OrificeC
+        level_pool_orifice_area: OrificeA
+#WRF-Hydro output file 
+forcing_parameters:
+    qlat_input_file: "../../test/input/florence_933020089/FORCING/SeptemberFlorenceQlateral.csv"
+    #qlat_input_folder: "../../test/input/florence_933020089/Full_Model_Run/OUTPUT/"
+    #qlat_file_pattern_filter: "/201809*.CHRTOUT_DOMAIN1"
+    #qlat_file_index_col: feature_id
+    #qlat_file_value_col: q_lateral
+#WRF-Hydro restart files
+restart_parameters:
+    ##WRF-Hydro channels restart file
+    #wrf_hydro_channel_restart_file: "../../test/input/CONUS_benchmark/RESTART/RESTART.2018110100_DOMAIN1.znc"
+    wrf_hydro_channel_restart_file: "../../test/input/florence_933020089/RESTART/HYDRO_RST.2018-09-01_00_00_DOMAIN1"
+    #WRF-Hydro channels ID crosswalk file
+    #wrf_hydro_channel_ID_crosswalk_file: "../../test/input/CONUS_benchmark/DOMAIN/RouteLink_NWMv2.1z.nc"
+    wrf_hydro_channel_ID_crosswalk_file: "../../test/input/florence_933020089/DOMAIN/Route_Link.nc"
+    wrf_hydro_channel_ID_crosswalk_file_field_name: link
+    wrf_hydro_channel_restart_upstream_flow_field_name: qlink1
+    wrf_hydro_channel_restart_downstream_flow_field_name: qlink2
+    wrf_hydro_channel_restart_depth_flow_field_name: hlink
+    #WRF-Hydro waterbodies restart file
+    wrf_hydro_waterbody_restart_file: "../../test/input/florence_933020089/RESTART/HYDRO_RST.2018-09-01_00_00_DOMAIN1"
+    #WRF-Hydro waterbody ID crosswalk file
+    wrf_hydro_waterbody_ID_crosswalk_file: "../../test/input/florence_933020089/DOMAIN/LAKEPARM.nc"
+    wrf_hydro_waterbody_ID_crosswalk_file_field_name: lake_id
+    #WRF-Hydro waterbody crosswalk filter file
+    wrf_hydro_waterbody_crosswalk_filter_file: "../../test/input/florence_933020089/DOMAIN/Route_Link.nc"
+    wrf_hydro_waterbody_crosswalk_filter_file_field_name: NHDWaterbodyComID
+parity_parameters:
+    parity_check_file: "../../test/input/florence_933020089/OUTPUT/SeptemberFlorenceStreamflow_wDates.csv" 
```

Comment:
Notice we are comparing against data in a .csv file (light), not observations within a series of CHRTOUT files (heavy)

Reply:
10-4. Good.